crates/biome_css_parser/src/syntax/parse_error.rs

```diff
@@ -7,6 +7,10 @@ pub(crate) fn expected_identifier(p: &CssParser, range: TextRange) -> ParseDiagn
     expected_node("identifier", range, p)
 }
 
+pub(crate) fn expected_express(p: &CssParser, range: TextRange) -> ParseDiagnostic {
+    expected_node("express", range, p)
```

Comment:
`expression`

What do you think if it's possible to have here examples of the css expressions?

Reply:
Do you mean to add the following?
```
pub(crate) fn expected_expression(p: &CssParser, range: TextRange) -> ParseDiagnostic {
    expected_any(&["component value", "binary expression", "parenthesized expression"], range, p)
}
```